main.js

```diff
@@ -0,0 +1,128 @@
+
+// Task #1
+
+// const createNewTeam = (nameOfTeam, nameOfDriver, dateBirthDriver, carBrand, teamSponsor, driversLicense) => {
+//     return {
+//         team: nameOfTeam,
+//         name: nameOfDriver,
+//         date: dateBirthDriver,
+//         car:  carBrand,
+//         sponsor: teamSponsor,
+//         license: driversLicense,
+//         isCreated: true,
+//     }
+// }
+
+// const newTeam = createNewTeam("Hillel", "Dmytro", 1992, "BMW", "Fil Dmytro", "confirm");
+
+// Task #2
+
+// let userInfo = {
+//     login: "",
+//     password: "",
+//     city: "",
+//     country: "",
+//     gender: "",
+//     age: "",
+// };
+
+// function createNewUser() {
+
+//     for (key in userInfo) {
+//         userInfo[key] = prompt(`Enter user ${key}...`);
+//     }
+
+//     return userInfo;
+// }
+// const newUser = (createNewUser());
+// console.log(newUser);
+
+
+
+
+// Task #3
+
+// function changeUserCity() {
+//     newUser.city = prompt("Enter new city");
+//     return newUser;

```

Comment:
And no need to return anything from this function. The purpose of the function - is to change data, not to produce value and return it as a result